frontend/pages/DashboardPage/cards/ActivityFeed/GlobalActivityItem/GlobalActivityItem.tests.tsx

```
@@ -1178,20 +1178,107 @@ describe("Activity Feed", () => {
+        host_display_name: "MacBookPro18,1 (ABCD)",
+        host_serial: "ABCD",
+      },
+    });
+    render(<GlobalActivityItem activity={activity} isPremiumTier />);
+
+    expect(
+      screen.getByText((_, node) => {
         return (
           node?.innerHTML ===
-          "<b>Test User </b>An end user turned on MDM features for a host with serial number <b>ABCD (manual)</b>."
+          "<b>MacBookPro18,1 (ABCD)</b> enrolled in Fleet."
```

ESLint

[error] Delete `⏎·········`

(prettier/prettier)